### PR TITLE
feat(liveness): Update BlazeFace version for face-detection

### PR DESCRIPTION
**Pull request by ayush987goyal**
#### Description of changes

 - Update the BlazeFace model to the latest FaceDetection model.
 - Update the implementation to adhere to new syntax.
 - Update the landmark calculation per the updated requirements.

#### Description of how you validated changes

- Validated locally and it's currently being also used in usability testing.

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by thaddmt — COMMENTED** [review]
LGTM so far, will approve after i test out locally

**Review by reesscot — CHANGES_REQUESTED** [review]
Temporarily blocking merge to main since we need to merge this to a mv feature branch

**Comment by thaddmt** [comment]
Going to include the changes in this PR on the next-release branch - https://github.com/aws-amplify/amplify-ui/pull/4685

**Pull request closed without merging**
